plugin/storage/kafka/options_test.go

```diff
@@ -28,12 +31,18 @@ func TestOptionsWithFlags(t *testing.T) {
 	command.ParseFlags([]string{
 		"--kafka.producer.topic=topic1",
 		"--kafka.producer.brokers=127.0.0.1:9092, 0.0.0:1234",
-		"--kafka.producer.encoding=protobuf"})
+		"--kafka.producer.encoding=protobuf",
+		"--kafka.producer.required-acks=local",
+		"--kafka.producer.compression=gzip",
+		"--kafka.producer.compression-level=6"})
```

Comment:
You need to change the value, as 6 is the default for gzip - so regardless of whether this flag is present, the test would currently pass.

Reply:
done.